### PR TITLE
#3: Code header and body (#4)

### PR DESCRIPTION
**Pull request by DatWithTheWorld**

### CODE REVIEW

**Review by xuanquocc — CHANGES_REQUESTED** [review]
End of lines

File: src/pages/index.html [hunk 1]
```diff
@@ -1,25 +1,31 @@
...
+    HTMLInspector.inspect();
+  </script>
+</body>
 
-    <script src="http://cdnjs.cloudflare.com/ajax/libs/html-inspector/0.8.2/html-inspector.js"></script>
-    <script>
-      HTMLInspector.inspect();
-    </script>
-  </body>
 </html>
```
Comment (xuanquocc) [inline src/pages/index.html:39]: end of line

**Review by xuanquocc — CHANGES_REQUESTED** [review]
You should create reset file

File: src/pages/index.html (hunk 1 above)
Comment (xuanquocc) [reply src/pages/index.html:39]: Have you fixed my comment yet?

File: src/stylesheets/abstracts/_variables.scss [hunk 2]
```diff
@@ -1 +1,20 @@
...
+//responsiev
+$mobile:429px;
+$medium: 768px;
+$desktop: 1200px;
+$tablet: calc($desktop - 1px);
+
+//font
+$primary-font : PoppinsRegular;
+$secondary-font : PoppinsBold;
+$third-font : MulishRegular;
```
Comment (xuanquocc) [inline src/stylesheets/abstracts/_variables.scss:23]: ?

File: src/stylesheets/layouts/_header.scss [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+            letter-spacing: -1px;
+        }
+        .header-main__desc{
+            text-align: center;
+            @include font();
+        }
+       }
+    }
+
+}
```
Comment (xuanquocc) [inline src/stylesheets/layouts/_header.scss:131]: ?

File: src/pages/index.html (hunk 1 above)
Comment (DatWithTheWorld) [reply src/pages/index.html:39]: > Have you fixed my comment yet?

image
I have already fixed in my lastest commit, sorry for my mistake.

File: src/stylesheets/abstracts/_variables.scss (hunk 2 above)
Comment (DatWithTheWorld) [reply src/stylesheets/abstracts/_variables.scss:23]: > ?

I saw in my design have two fonts: Mulish and Poppins, but Poppins font not enough to work, so i must use PoppinsRegular and PoppinsBold

File: src/pages/index.html (hunk 1 above)
Comment (xuanquocc) [reply src/pages/index.html:39]: i don't see your resolve in your pr i don't see end of line

File: src/layouts/header.html [hunk 4]
```diff
@@ -1,3 +1,48 @@
...
+                Build beautiful landing pages in record time with <a href="" class="desc__link">Anima</a>’s Landing Page UI kit for Figma. No code required!
+            </p>
+        </div>
+        <div class="header-main__button">
+            <button class="button-getst"><img src="../assets/icons/ic-rocket.svg" alt="rocket" class="icon-getst">Get Started</button>
+            <button class="button-hiw">How it works</button>
+        </div>
+        <div class="header-main__picture">
+            <div class="header-main__picture--block">
+                <img src="../assets/icons/ic-substract.svg" alt="Star" class="icon-star icon-star--white">
```
Comment (xuanquocc) [inline src/layouts/header.html:39]: i think you shoud figure tag to wrap img

File: src/stylesheets/layouts/_header.scss [hunk 5]
```diff
@@ -24,8 +29,37 @@
                 display: none;
             }
             .getst{
-                @include button($width:220px,$height:60px,$background-color:$button-color-primary,$border-radius:20px);
+                @include button($width:220px,$height:60px,$background-color:$button-color-primary,$border-radius:20px,$border:none);
```
Comment (xuanquocc) [inline src/stylesheets/layouts/_header.scss:36]: you should not use px to style. it so difficult to flexible

File: src/pages/index.html (hunk 1 above)
Comment (DatWithTheWorld) [reply src/pages/index.html:39]: > i don't see your resolve in your pr i don't see end of line

I resolve it at https://github.com/DatWithTheWorld/practice_html_css/pull/4/commits/3396a2442d5a10396203a4ccb9ca50f7092eaadf

File: src/layouts/header.html [hunk 6]
```diff
@@ -1,3 +1,48 @@
-<header>
-    dafjsdf
-</header>
\ No newline at end of file
+<header class="header-main">
+    <div class="container">
+      <!-- Start of header-top -->
+      <div class="header-main__top">
+        <a class="header-main__logo">
+            <img src="../assets/icons/ic-television.svg" alt="television" class="logo">
```
Comment (baonguyen26) [inline src/layouts/header.html:6]: You should wrap them with the `figure`

File: src/layouts/header.html [hunk 7]
```diff
@@ -1,3 +1,48 @@
...
+      <div class="header-main__bottom">
+        <!-- Start of header content -->
+        <div class="header-main__content">
+            <h1 class="header-main__title">Create Engaging Landing Pages</h1>
+            <p class="header-main__desc">
+                Build beautiful landing pages in record time with <a href="" class="desc__link">Anima</a>’s Landing Page UI kit for Figma. No code required!
+            </p>
+        </div>
+        <div class="header-main__button">
+            <button class="button-getst"><img src="../assets/icons/ic-rocket.svg" alt="rocket" class="icon-getst">Get Started</button>
```
Comment (baonguyen26) [inline src/layouts/header.html:34]: the same, and pls format your code

File: src/layouts/header.html (hunk 6 above)
Comment (DatWithTheWorld) [reply src/layouts/header.html:6]: I'll repair it before my report at tomorrow. Thank you !

**Pull request merged**